Pal/src/host/Linux-SGX/db_main.c

```diff
@@ -393,11 +393,6 @@ void pal_linux_main(char * uptr_args, uint64_t args_size,
     __pal_control.manifest_preload.start = (PAL_PTR) manifest_addr;
     __pal_control.manifest_preload.end = (PAL_PTR) manifest_addr + manifest_size;
 
-    if ((rv = init_trusted_platform()) < 0) {
```

Comment:
I am fine(and agree) with this change. But still curious, if the removal of init_trusted_platform, which was previously used to initiate RA with IAS, upon startup, has been discussed with @chiache and others in graphene core team.